providers/class-two-factor-totp.php

```diff
@@ -314,7 +314,11 @@ public function user_two_factor_options( $user ) {
 				<input type="hidden" id="two-factor-totp-key" name="two-factor-totp-key" value="<?php echo esc_attr( $key ); ?>" />
 				<label for="two-factor-totp-authcode">
 					<?php esc_html_e( 'Authentication Code:', 'two-factor' ); ?>
-					<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9]*" />
+					<?php
+						/* translators: Example auth code. */
+						$placeholder = sprintf( __( 'eg. %s', 'two-factor' ), '123456' );
```

Comment:
I've taken a slightly different approach, which I'd like your review on @iandunn.

Instead of treating it like a string number, or groups of two digits, I've instead increased the width between all the characters, which both makes it easier to read, and due to the increased space, "feels like" (to me) that there's less of a requirement for spaces on the input (as it's auto-spaced).

Adding a space between the chunks like this is still possible, and ends up like this:
| Current | with extra space | user input |
| --- | --- | --- |
| <img width="332" src="https://user-images.githubusercontent.com/767313/218906186-45e85e68-29ef-49f9-9433-05b3499f81d1.png"> | <img width="332" alt="Screenshot 2023-02-15 at 11 52 57 am" src="https://user-images.githubusercontent.com/767313/218907367-d0332b5a-4469-4db1-8e48-008268cbeae3.png"> | <img width="331" alt="Screenshot 2023-02-15 at 11 55 46 am" src="https://user-images.githubusercontent.com/767313/218907549-5f3b84a1-e6c5-4045-8ac7-52a1731cdf02.png"><br>(no user-input spaces)|

Reply:
🤔 , I like this better than `e.g., 123 456`. I lean towards `123 456` being best, though. IMO blocks of numbers like in auth codes have two problems:

1) they're hard to _read_ because the numbers all run together
2) they're hard to _remember_ because the sequence is too long

Adding extra (uniform) space between each letter solves `1`, but not `2`. _Grouping_ the numbers into small blocks like `123 456` solves both problems IMO.

I don't feel strongly, though. I think `1 2 3 4 5 6` is an improvement on the current situation 👍🏻